spring-web/src/main/java/org/springframework/web/service/invoker/HttpServiceProxyFactory.java

```diff
@@ -264,10 +268,18 @@ private HttpServiceMethodInterceptor(List<HttpServiceMethod> methods) {
 		}
 
 		@Override
+		@SuppressWarnings({"unchecked"})
 		public Object invoke(MethodInvocation invocation) throws Throwable {
 			Method method = invocation.getMethod();
 			HttpServiceMethod httpServiceMethod = this.httpServiceMethods.get(method);
 			if (httpServiceMethod != null) {
+				if (KotlinDetector.isSuspendingFunction(method)) {
+					Object[] arguments = getSuspendedFunctionArgs(invocation.getArguments());
+					Continuation<Object> continuation = resolveContinuationArgument(invocation.getArguments());
```

Comment:
`kotlin.coroutines.Continuation` may not be in the classpath, so any usage of this class should probably be isolated in a `KotlinDelegate` inner class (see for example `BeanUtils` for an example of such pattern) to avoid classpath errors on non Kotlin projects.